Project/frontend/package.json

```diff
@@ -24,6 +24,7 @@
     "react-filepond": "^7.1.2",
     "react-router-dom": "^6.23.1",
     "react-sigma": "*",
+    "react-toastify": "^10.0.5",
```

Comment:
I wanted to let you know that we already use the React Material UI library in our codebase. We also utilize the Snackbar component to display status messages (Success, Error, Info, Warning, etc.), which you can find here: https://mui.com/material-ui/react-snackbar/.
It would be wonderful if you could use Material UI for this purpose as well. This approach would ensure consistency within our codebase and help reduce our build size since we wouldn't need an additional package to achieve it.